website/docs/r/dashboard.html.markdown

```diff
@@ -272,7 +272,7 @@ The following arguments are supported:
 - `description` - (Optional) Description of the dashboard.
 - `folder_id` - (Optional) The identifier of the folder to save the dashboard in. By default it is saved in your
 personal folder.
-- `refresh_interval` - (Optional) Interval of time (in seconds) to automatically refresh the dashboard.
+- `refresh_interval` - (Optional) Interval of time (in seconds) to automatically refresh the dashboard. Valid values are 120,300,900,1800,3600,7200,86400.
```

Comment:
We should add the validation in the resource as well.

Reply:
Ohh ya, I'll add that